sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOTest.java

```diff
@@ -845,13 +842,11 @@ public void testBuildWrite() {
   }
 
   @Test
-  @Ignore("[BEAM-436] DirectRunner tempLocation configuration insufficient")
   public void testBatchWritePrimitiveDisplayData() throws IOException, InterruptedException {
```

Comment:
Deleted these tests and create https://issues.apache.org/jira/browse/BEAM-2167 to figure out the appropriate display data.